app/Repositories/Eloquents/WebhookRepository.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace App\Repositories\Eloquents;
+ 
+use App\Models\Webhook;
+use App\Repositories\Interfaces\WebhookRepositoryInterface;
+use Auth;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/WebhookRepository.php</br>**Line**: 8</br>Whitespace found at end of line